.urcheon/ignore.txt

```diff
@@ -0,0 +1,2 @@
+source.txt
+*.blend
```

Comment:
Urcheon should already ignore `.blend` files. If the builtin ignore doesn't work this is a bug to report, otherwise this line can be removed.

Reply:
done